docs/config_search.md

```diff
@@ -18,84 +18,132 @@ limitations under the License.
-that are automatically searched are
-[`max_batch_size`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size)
-and
-[`instance_group`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups).
-Additionally, [`dynamic_batching`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher) will be enabled if it is legal to do so.
+- [Automatic](config_search.md#automatic-brute-search)
+  - **No** model config parameters are specified
+- [Manual](config_search.md#manual-brute-search)
+  - **Any** model config parameters are specified
+  - `--run-config-search-disable` option is specified
 
-An example model analyzer config that performs automatic config search looks
-like below:
+---
+
+## Automatic Brute Search
+
+**Default brute search mode when no model config parameters are specified**
+
+The parameters that are automatically searched are
+[model maximum batch size](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size),
+[model instance groups](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups), and [request concurrencies](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/perf_analyzer.md#request-concurrency).
+Additionally, [dynamic_batching](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher) will be enabled if it is legal to do so.
+
+_An example model analyzer YAML config that performs an Automatic Brute Search:_
 
 ```yaml
 model_repository: /path/to/model/repository/
 
 profile_models:
-  - model_1
-  - model_2
+  - model_A
 ```
 
-In the default mode, automatic config search will try values 1 through 5 for
-[`instance_group`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups).
-The maximum value can be changed using the `run_config_search_max_instance_count` key in the Model Analyzer Config.
-For each `instance_group`, Model Analyzer will sweep values 1 through 128 increasing exponentially (i.e. 1, 2, 4, 8, ...) for
-[`max_batch_size`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size). The start and end values can be changed using `run_config_search_min_model_batch_size` and `run_config_search_max_model_batch_size`.
-[`Dynamic_batching`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher)
-will be enabled for all model configs generated using automatic search.
+You can also modify the minimum/maximum values that the automatic search space will iterate through:
+
+---
```

Comment:
again, the `through:` implies there will be something after it, but then the thick line implies that what is below is not related to this.

Reply:
removed the line